frontend/lab/samples/intro/welcome.htmlf

```diff
@@ -8,10 +8,15 @@
   Hit <strong>Run</strong> on the right 👉. <br />
   That's what we'll build up to in this first lab—a bit of a whirlwind tour!
 </p>
-<p>
-  There is also a step-by-step
-  <a href="https://youtu.be/FBqyiU4iZNY?si=9vFayb_R2rAgEMbX" target="_blank">video guide</a> if you
-  need help..
-</p>
+<p>There is also a step-by-step video guide:</p>
+<iframe
+  src="https://www.youtube.com/embed/FBqyiU4iZNY?si=9vFayb_R2rAgEMbX"
```

Comment:
Is the `si` query parameter needed? I usually remove query params on youtube links and they work just fine. Or is this some metrics param for tracking watches or something?

Reply:
yeah, you are right, removed. thank you!